vedro_jira_failed_reporter/_failed_reporter_plugin.py

```diff
@@ -183,10 +185,9 @@ def on_scenario_failed(self, event: Union[ScenarioPassedEvent, ScenarioFailedEve
             self._reporting_language.ISSUE_CREATED.format(jira_server=self._jira_server, issue_key=result_issue.key)
         )
 
-        traceback = event.scenario_result._step_results[-1].exc_info.traceback
```

Comment:
i suppose we don't need whole issues linking section anymore.

if we will make single issue per file we won't get any separate related issues. So no need to search and link any at all.

Reply:
Code for creating links was removed.